### DISCUSSION TITLE
Adjusting tempo aka playback speed:

### DISCUSSION THREAD

**Discussion by NedH13** [z-huang/music#1252] [Ideas]
Hello, 
Can you please make the playback speed increments to be .05 instead of .25 please?
0.25 makes more sense for videos. Music however need .05 increments. 

I need this feature, it is mandatory for my usage, the app is not usable without it so I hope it can get added asap ☺️ 
Thanks so much for your effort ✌️

![Screenshot_20240321-151655~2](https://github.com/z-huang/InnerTune/assets/77459978/81180ef6-b153-425b-b83e-8f99296398a1)

**Comment by Malopieds** [comment]
Hi, it has been implemented in [this](https://github.com/Malopieds/InnerTune/commit/203b163ae275320069f21f5d2ee0019c8a8b3eec) commit. This will be part of the next release.

**Reply by NedH13** [reply]
Oh! Thank you so much for forking it :) such a life saver
When to expect it?

**Reply by Malopieds** [reply]
I think in a week or two!

**Comment by fanieko** [comment]
I think speed playback should be a slider..

**Reply by Malopieds** [reply]
I agree with that, I'll try to implement it asap.
